src/pages/blocks/_components/BlockRow.tsx

```diff
@@ -0,0 +1,149 @@
+        <div className="grid grid-cols-8 lg:grid-cols-12 gap-5 py-4">
+          <div className="col-start-1 col-end-3">
+            <BlockNumberComponent number={data.number} />
+          </div>
+          <div className="col-start-3 col-end-7 lg:col-start-3 lg:col-end-9">
+            <div className="flex md:flex-col lg:flex-row">
+              <RewardComponent
+                amount={data.extraData?.reward ?? "0"}
+                symbol={data.extraData?.symbol ?? ""}
+                containerClass="lg:w-1/2"
+              />
+              <RecipientComponent
+                value={data.miner}
+                containerClass="ml-0 md:mt-4 lg:ml-2.5 lg:mt-0"
+              />
+            </div>
+          </div>
+          <div className="col-start-7 col-end-9 lg:col-start-9 lg:col-end-13">
+            <div className="flex flex-col lg:flex-row justify-between">
+              <TxnCountComponent
+                count={data.transactions.length}
+                containerClass="w-full text-start md:text-right lg:text-start lg:w-1/2"
+              />
+              <TimeComponent
+                time={data.timestamp}
+                containerClass="text-right md:mt-4 lg:mt-0"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* For mobile */}
+      <div className="md:hidden py-6">
+        <BlockNumberComponent number={data.number} />
+        <div className="ml-8">
+          <RewardComponent
+            amount={data.extraData?.reward ?? "0"}
+            symbol={data.extraData?.symbol ?? ""}
+            containerClass="mt-2"
+          />
+          <RecipientComponent value={data.miner} containerClass="mt-4" />
+          <TxnCountComponent
+            count={data.transactions.length}
+            containerClass="mt-4"
+          />
+          <TimeComponent time={data.timestamp} containerClass="mt-4" />
+        </div>
+      </div>
+      <div className="border-b border-black-600 ml-8 lg:ml-10" />
+    </div>
+  );
+}
+
+function BlockNumberComponent({ number }: { number: string }): JSX.Element {
+  return (
+    <div className="flex flex-row">
+      <FiBox size={24} className="text-white-50 stroke-white-50" />
+      <LinkText
+        testId={`block-number-${number}`}
+        href={`/blocks/${number}`}
+        customStyle="ml-2 lg:ml-4"
+      >
+        <NumericFormat
+          thousandSeparator
+          value={number}
+          decimalScale={0}
+          prefix="#"
+        />
+      </LinkText>
+    </div>
+  );
+}
+
+function RewardComponent({
+  amount,
+  symbol,
+  containerClass,
+}: {
+  amount: string;
+  symbol: string;
+  containerClass?: string;
+}): JSX.Element {
+  return (
+    <div className={clsx("flex flex-row", containerClass)}>
+      <span className="text-white-700 text-base mr-1">Reward</span>
+      <NumericFormat
+        className="text-white-50 text-base"
+        thousandSeparator
+        value={new BigNumber(amount).toFixed(8)}
```

Comment:
do we need to convert before passing to value? because `NumericFormat` will `toFixed(decimalScale)`